### PR TITLE
New CDWS Type

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
@kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/CDWS.yaml [hunk 1]
```diff
@@ -16,6 +16,13 @@
 ### Canonical Types  ###
 ########################
 
+CDWS_CWDT:
+  description: "Condenser water system with temperature differential monitoring across condenser water."
+  is_canonical: true
+  implements:
+  - CDWS
+  - CWDT
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/CDWS.yaml:24]: No need to specify that this is condensing water, use WDT

File: ontology/yaml/resources/HVAC/entity_types/CDWS.yaml [hunk 2]
```diff
@@ -16,6 +16,13 @@
 ### Canonical Types  ###
 ########################
 
+CDWS_CWDT:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/CDWS.yaml:19]: Please confirm there is no temperature control on this loop

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/CDWS.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @cstirdivant - thank you, changed to WDT there. Will confirm re temperature control ASAP.

Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @Ciaran-Oflannagain any update on if there is supply (or return) water temperature control on this system? How does it know what temperature of water to provide to the chillers?

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @cstirdivant - Apologies still awaiting clarity from BMS Contractor.

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @cstirdivant - I can confirm there is no temperature control on this packaged system

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @cstirdivant - is there any further information needed for this CDWS type? Thanks, Ciaran

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/CDWS.yaml]: @cstirdivant - just following up on this PR. Thanks

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**